app/models/stats.rb

```diff
@@ -0,0 +1,120 @@
+# typed: false
+
+class Stats
+  FIRST_MONTH = Time.new(2012, 7, 3).utc.freeze
+  TIMESCALE_DIVISIONS = "1 year".freeze
+  USERS_GRAPH_CACHE_KEY = "users_graph".freeze
+  ACTIVE_USERS_GRAPH_CACHE_KEY = "active_users_graph".freeze
+  STORIES_GRAPH_CACHE_KEY = "stories_graph".freeze
+  COMMENTS_GRAPH_CACHE_KEY = "comments_graph".freeze
+  VOTES_GRAPH_CACHE_KEY = "votes_graph".freeze
+
+  def self.get_users_graph = get_cached_graph(USERS_GRAPH_CACHE_KEY)
+
+  def self.get_active_users_graph = get_cached_graph(ACTIVE_USERS_GRAPH_CACHE_KEY)
+
+  def self.get_stories_graph = get_cached_graph(ACTIVE_USERS_GRAPH_CACHE_KEY)
+
+  def self.get_comments_graph = get_cached_graph(COMMENTS_GRAPH_CACHE_KEY)
+
+  def self.get_votes_graph = get_cached_graph(VOTES_GRAPH_CACHE_KEY)
```

Comment:
Ruby style for this kind of key is just to make these keys symbols, they're immutable strings. The coupling of a shared key is the same whether it's a constant (`USERS_GRAPH_CACHE_KEY`) or symbol (`:users_graph`). (My favorite introduction to Ruby for experienced programmers is [The Well-Grounded Rubyist](https://www.manning.com/books/the-well-grounded-rubyist-third-edition), it covers lots of Ruby style.)

The `fill_` methods have different queries so let's leave them alone, but we can delete these constants and the individual `get_` methods.